File: aurora-scala/src/main/scala/net/gree/aurora/scala/domain/cluster/TaggedClusterIdResolverIdImpl.scala
package net.gree.aurora.scala.domain.cluster

import net.gree.aurora.domain.cluster.{TaggedClusterIdResolverId => JTaggedClusterIdResolverId}

class TaggedClusterIdResolverIdImpl(val underlying: JTaggedClusterIdResolverId) extends TaggedClusterIdResolverId {
  def value: Any = underlying.getValue
}

File: aurora-core/src/main/java/net/gree/aurora/domain/cluster/TaggedClusterIdResolverIdImpl.java
package net.gree.aurora.domain.cluster;

import org.sisioh.dddbase.model.impl.AbstractIdentity;

final class TaggedClusterIdResolverIdImpl extends AbstractIdentity<Object> implements TaggedClusterIdResolverId {

    public TaggedClusterIdResolverIdImpl(Object value) {
        super(value);
    }

    @Override
    public String toString() {
        return String.format("TaggedClusterIdResolverId(value = %s)", value);
    }

}
